code/game/objects/items/explosives/grenades/marines.dm

```diff
@@ -465,6 +465,172 @@
 	icon_state = "grenade_phos_clf"
 	item_state = "grenade_phos_clf"
 
+/obj/item/explosive/grenade/sebb
+	name = "\improper G2 Electroshock grenade"
+	desc = "This is a G2 Electroshock Grenade. Produced by Armat Battlefield Systems, it's sometimes referred to as the Sonic Electric Ball Breaker, \
+		after a rash of incidents where the intense 1.2 gV sonic payload caused... rupturing. \
+		A bounding landmine mode is available for this weapon which activates a small drill to self-bury itself when planted. Simply plant it at your feet and walk away."
+	icon_state = "grenade_sebb"
+	item_state = "grenade_sebb"
+	/// Maximum range of effect
+	var/range = 5
+	/// Maximum possible damage before falloff.
+	var/damage = 110
+	/// Factor to mutiply the effect range has on damage.
+	var/falloff_dam_reduction_mult = 20
+	/// Post falloff calc damage is divided by this to get xeno slowdown
+	var/xeno_slowdown_numerator = 12
+	/// Post falloff calc damage is multipled by this to get human stamina damage
+	var/human_stam_dam_factor = 0.9
+
+/obj/item/explosive/grenade/sebb/get_examine_text(mob/user)
+	. = ..()
+	. += SPAN_NOTICE("To put into mine mode, plant at feet.")
+
+/obj/item/explosive/grenade/sebb/afterattack(atom/target, mob/user, proximity)
+	var/turf/user_turf = get_turf(user)
+	if(active)
+		return
+
+	if(!isturf(target))
+		return
+
+	if(user.action_busy)
+		return
+
+	if(target != get_turf(user))
+		return
+
+	if(locate(/obj/item/explosive/mine) in get_turf(src))
+		to_chat(user, SPAN_WARNING("There already is a mine at this position!"))
+		return
+
+	if(user_turf && (user_turf.density || locate(/obj/structure/fence) in user_turf))
+		to_chat(user, SPAN_WARNING("You can't plant a mine here."))
+		return
+
+	if(Adjacent(/obj/item/explosive/mine)) // bit more strict on this than normal mines
+		to_chat(user, SPAN_WARNING("Too close to another mine! Plant it somewhere less obvious."))
+		return
+
+	user.visible_message(SPAN_NOTICE("[user] starts deploying [src]."),
+		SPAN_NOTICE("You switch [src] into landmine mode and start placing it..."))
+	playsound(user.loc, 'sound/effects/thud.ogg', 40)
+	if(!do_after(user, 5 SECONDS * user.get_skill_duration_multiplier(SKILL_CONSTRUCTION), INTERRUPT_ALL|BEHAVIOR_IMMOBILE, BUSY_ICON_BUILD))
+		to_chat(user, SPAN_NOTICE("You stop planting."))
+		return
+
+	user.visible_message(SPAN_NOTICE("[user] finishes deploying [src]."),
+		SPAN_NOTICE("You finish deploying [src]."))
+	var/obj/item/explosive/mine/sebb/planted = new /obj/item/explosive/mine/sebb(get_turf(user))
```

Comment:
See `/obj/item/explosive/mine/Initialize()` where it for whatever reason is calling `deploy_mine(null)` because it has map_deployed true. This results in these basically having their IFF hardcoded to USCM. You likely want to do it more like how `/obj/item/explosive/mine/attack_self(mob/living/user)` does it.

Reply:
IFF is set when planting now.